Add disclaimer that only github authentication is supported (#20)
redhat-ai-dev/ai-rhdh-installer: README.md

# Red Hat Developer Hub Installer for AI Software Templates

> > We are only supporting **GitHub Authentication**, **GitLab Authentication** is planned for future versions.

Reviewer: I'm wondering if this should be marked as important same as what @johnmcollier did on the template repo here https://github.com/redhat-ai-dev/ai-lab-template/commit/4ec1c75ff3cb0cd1827bb454b8a4155a534092f9?

Thoughts?

Reviewer 2: yeah sure that makes sense

Author: addressed in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/20/commits/e41515d7a51f655a8160c0143eb35e0b273d007a

> > We are only supporting **GitHub Authentication**, **GitLab Authentication** is planned for future versions.

Reviewer 2: ```suggestion
> Currently, only the **GitHub Authentication** is supported. **GitLab Authentication** is planned for future versions.
```

Detailed documentation for configuring Developer Hub can be found in [`RHDH-CONFIG.md`](./docs/RHDH-CONFIG.md).

## GitLab Integration

> To configure RHDH to use GitLab as the git repository source, you will need to first run `export RHDH_GITLAB_INTEGRATION=true` before running any of the [configuration](#configuration) scripts. Run `export RHDH_GITHUB_INTEGRATION=false` to disable GitHub integration.

Reviewer 2: If we are going to put this section and other env config sections below; I think we can add a line before bash configure.sh to say; please scroll below to customize your configuration if you dont want the default

Reviewer 2: otherwise many would just run configure.sh and then wonder how to customize things..

Author: @maysunfaisal https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/20/commits/25901016bd3d563de50dad953689dd8c4984f143 should resolve this, let me know what you think.
